core/Enclave/key_operation.cpp

```diff
@@ -701,6 +701,7 @@ sgx_status_t ehsm_rsa_decrypt(const ehsm_keyblob_t *cmk,
                               ehsm_data_t *plaintext)
 {
     sgx_status_t ret = SGX_ERROR_UNEXPECTED;
```

Comment:
commit msg
"
Return a fixed length of plaintext for RSA decryption

Previously, when the input of datalen for the plaintext is zero, it will
calculate the actual size of the plaintext buffer by performing one extra 
time of rsa decryption. However, we can return a fixed buffer size directly
to the user to reduce unnecessary calculations.
"